specification/cognitiveservices/OpenAI.Inference/models/completions/azure_chat_extensions.tsp

```diff
@@ -229,3 +229,29 @@ model AzureGroundingEnhancementCoordinatePoint {
   @doc("The y-coordinate (vertical axis) of the point.")
   y: float32;
 }
+
+/**
+ * User security context contains several parameters that describe the AI application itself,
+ * and the end user that interacts with the AI application. These fields assist your security
+ * operations teams to investigate and mitigate security incidents by providing a comprehensive
+ * approach to protecting your AI applications. (Learn more at https://aka.ms/TP4AI/Documentation/EndUserContext)
+ * about protecting AI applications using Microsoft Defender for Cloud.
+ */
+@added(ServiceApiVersions.v2024_02_15_Preview)
```

Comment:
Why so early? Depending on the tracking of private releases, I believe this was only formally supported with `2024-11-01-preview` (PrPr) with `2024-12-01-preview` being the better, public start date. Do things get weird if we don't backport this to the beginning of time?

Reply:
fixed, autocomplete miss XP